onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -64,7 +64,7 @@ BackendManager::BackendManager(const GlobalContext& global_context,
     i++;
   }
   subgraph_context_.subgraph_name = fused_node.Name();
-  model_proto_ = GetModelProtoFromFusedNode(fused_node, subgraph, logger);
```

Comment:
Please leave this line here and manage the result as needed. Replicating the call results in spaghetti code as it isn't clear why there are multiple calls. Here's the original code for reference https://github.com/intel-sandbox/jemartin-onnxruntime/commit/792523185d156550165eab34c92184b25bf09267